openml-caret/README.md

```diff
@@ -0,0 +1,34 @@
+OpenML Provider for caret models
+------------------------------------------------------------
+
+This module contains an [OpenML provider](https://github.com/feedzai/feedzai-openml) for loading models trained with the [Caret package](http://caret.r-forge.r-project.org/).
+
+The implemented "Classifier" should define the following methods: 
+
+```caret
```

Comment:
this should be replaced by `r`.

This tells the markdown this piece of code is written in a certain language (R in this case) so that it knows how to properly do the syntax highlight. Java, Python, and R are programming languages. scikit and caret are libraries/packages/frameworks for python and r.